src/components/QrView.tsx

```diff
@@ -51,7 +50,7 @@ const QrComponent = ({ navigateForward }: Props) => {
 		<View style={styles.container}>
 			<BarCodeScanner
 				onBarCodeScanned={
-					scanned
+					scanned || state.bot
```

Comment:
Actually, why do we need `scanned` in the first place? If we don't have this flag, will the barcode scanner just scan continuously?